src/blockchain_txn_add_subnetwork_v1.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package helium;
+
+import "blockchain_token_type_v1.proto";
+
+message blockchain_txn_add_subnetwork_v1 {
+  blockchain_token_type_v1 token_type = 1;
+
+  bytes subnetwork_key = 2;
+  repeated bytes reward_server_keys = 3;
```

Comment:
should this be a list of keys or a 1/N multisig (where N can start as 1 for now) but can be changed later if we want to add / migrate keys.

Reply:
no, a list is more appropriate until we change the model to one where we have multiple servers double checking each other, which is not what we have right now.